src/utils.h

```diff
@@ -19,6 +19,9 @@
 #include <nodes/relation.h>
 #endif
 
+#define MAX(x, y) ((x) > (y) ? x : y)
+#define MIN(x, y) ((x) < (y) ? x : y)
+
```

Comment:
Nit: These functions evaluate the parameters twice, so need to be used with care.